integration_tests/integration/view-recall.spec.js

```diff
@@ -81,6 +81,9 @@ context('View a recall', () => {
     })
     recallInformation.assertElementHasText({ qaAttr: 'dossierEmailSentDate', textToFind: '8 September 2021' })
     recallInformation.assertElementHasText({ qaAttr: 'dossierCreatedByUserName', textToFind: 'Bobby Badger' })
+    recallInformation.assertElementHasText({ qaAttr: 'bookedByUserName', textToFind: 'Brenda Badger' })
+    recallInformation.assertElementHasText({ qaAttr: 'assessedByUserName', textToFind: 'Bertie Badger' })
+
     recallInformation.assertElementHasText({ qaAttr: 'recallStatus', textToFind: 'Dossier issued' })
```

Comment:
Can we assert for dossierCreatedByUserName somewhere?

Reply:
Looks like it was done before on line 83

Reply:
sorry I missed that